CMakeLists.txt

```diff
@@ -92,5 +92,4 @@ if(("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang") OR
 endif()
 
 add_subdirectory(src)
-
-install(FILES LICENSE DESTINATION .)
```

Comment:
```
-- Installing: /usr/python/artm/library.py
-- Installing: /usr/python/artm/artm_model.py
-- Installing: /usr/python/artm/messages_pb2.py
-- Installing: /usr/python/artm/__init__.py
```
This is incorrect behaviour too. Because python in the user's environment could store packages not in `<install-prefix>/python`. After installation user needs to add this dir to PYTHONPATH, but this is a minor problem.

The main issue is that we've started to use python dependencies like `numpy` and `pandas`. Before installing we need to check them and install if it is needed. Dependency management in python is done with [PyPI](https://pypi.python.org/pypi) and `setup.py` scripts based on [distutils or setuptools](http://stackoverflow.com/questions/6344076/differences-between-distribute-distutils-setuptools-and-distutils2).